RadixWallet/Clients/NPSSurveyClient/NPSSurveyClient+Live.swift

```diff
@@ -101,51 +103,37 @@ extension NPSSurveyClient {
 			}
 		}()
 
-		var urlComponents = URLComponents(string: Self.rootURL)!
-		urlComponents.queryItems = [.userId(userId), .formUUID] + (feedback.map(\.queryItems) ?? [])
+		let urlComponents = URLComponents(string: Self.rootURL)!
 
 		guard let url = urlComponents.url else {
 			return
 		}
 
 		var urlRequest = URLRequest(url: url)
 		urlRequest.httpMethod = "POST"
+		urlRequest.allHTTPHeaderFields = [
+			"accept": "application/json",
+			"Content-Type": "application/json",
+		]
+
+		var feedbackParams: [String: Any] = [
+			BodyParam.id.rawValue: userId.uuidString,
+			BodyParam.formUuid.rawValue: formUUID,
+		]
+		if let feedback {
+			feedbackParams[BodyParam.nps.rawValue] = feedback.npsScore
+		}
+		if let reason = feedback?.reason {
+			feedbackParams[BodyParam.feedbackReason.rawValue] = reason
+		}
+		urlRequest.httpBody = try? JSONSerialization.data(withJSONObject: feedbackParams)
 
 		do {
-			_ = try await httpClient.executeRequest(urlRequest)
+			_ = try await httpClient.executeRequest(urlRequest) { statusCode in
+				[200, 202].contains(statusCode)
```

Comment:
This request returns a 202 response code. For other requests, we want to be more restrictive and accept only a 200 response code.